lib/api/jobRouter.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const router = express.Router();
+
+const jobController = require("../dbConnections/jobController");
+const { Status } = require("../constants/responseAndStatus");
+const logger = require("../logger/log");
+
+router.post("/create", (req, res) => {
+    if (!req.body) {
+        return res.status(400).json({
+            "error": "Bad Request"
+        });
+    } else {
+        jobController.createJob(req.body.id, req.body.description, req.body.frequency, req.body.script)
+            .then(response => {
+                if (response.data !== null) {
+                    res.location = response.data;
```

Comment:
Possibly untested- response does not have a field called "location". Even if you attach this custom property to it, it would not have any effect on the HTTP response. You're probably looking for `res.set('Location', ...)`

Reply:
Also, why is the "data" field used in the location header?